benchmarks/benchmark/tools/dlio/modules/dlio/podspec.tpl

```diff
@@ -67,11 +67,18 @@ spec:
           mountPath: ${dlio_data_mount_path}
         - name: dshm
           mountPath: /dev/shm
+        - name: results
+          mountPath: /dlio_results
       serviceAccountName: ${service_account}
       volumes:
       - name: ml-perf-volume
         persistentVolumeClaim:
           claimName: ${pvc_name}
+      - name: results
+        csi:
+          driver: gcsfuse.csi.storage.gke.io
```

Comment:
is this job.tf shared cross different drivers? if so this should be a variable?

Reply:
the idea here is that we are going to write result data to a second gcs bucket, no matter what kind of storage we run the benchmark on. so the gcsFuse should always be true IIUC

Reply:
I see, make sense